.editorconfig

```diff
@@ -1,36 +0,0 @@
-# RoboCup base editorconfig
```

Comment:
Is there any reason files like this and your setup script were removed?

Reply:
@ebretl was just cleaning out most of everything to get his simple offline script working. We’ll be adding back in some of this infrastructure stuff after competition.
They didn’t need to be removed now, but most of the team doesn’t understand the parts of RC’s system that we borrowed, so I’m not going to worry about it until I have time after AVC to go through it as a team and get everyone up to speed